discovery-provider/plugins/notifications/src/processNotifications/mappers/usdcPurchaseBuyer.ts

```diff
@@ -0,0 +1,169 @@
+import { Knex } from 'knex'
+import { NotificationRow } from '../../types/dn'
+import {
+  USDCPurchaseBuyerNotification,
+  AppEmailNotification
+} from '../../types/notifications'
+import { BaseNotification } from './base'
+import { sendPushNotification } from '../../sns'
+import { ResourceIds, Resources } from '../../email/notifications/renderEmail'
+import { sendNotificationEmail } from '../../email/notifications/sendEmail'
+import {
+  buildUserNotificationSettings,
+  Device
+} from './userNotificationSettings'
+import { disableDeviceArns } from '../../utils/disableArnEndpoint'
+import { capitalize } from 'lodash'
+import { sendBrowserNotification } from '../../web'
+import { EntityType } from '../../email/notifications/types'
+
+type USDCPurchaseBuyerRow = Omit<NotificationRow, 'data'> & {
+  data: USDCPurchaseBuyerNotification
+}
+export class USDCPurchaseBuyer extends BaseNotification<USDCPurchaseBuyerRow> {
+  notificationReceiverUserId: number
+  sellerUserId: number
+  amount: number
+  content_id: number
```

Comment:
Wondering again about the mixture of casing 😅

Reply:
ah yea that's just accidental lol will fix